src/core/uuid.ts

```diff
@@ -0,0 +1,8 @@
+export function simpleUuid() {
```

Comment:
Is there no any available library to do this job? If we have some really special reasons to write our own version, I think a comment should be put here